101-batch-with-keyvault-pfx-password/README.md

```diff
@@ -0,0 +1,11 @@
+# ARM template for using Azure Batch service with pfx password certificate from keyvault
```

Comment:
Shorten the title

Azure Batch Service with Certificate from KeyVault

Reply:
Done